artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/browse.js

```diff
@@ -343,7 +352,7 @@ var Artemis;
             origQueue = {
                 itemField: 'StringProperties',
                 header: 'Original Queue',
-                templateFn: function(value) {
+                templateFn: function(value) {next
```

Comment:
Is the `next` word on the 355 line a typo?

Reply:
It is Ive fixed it, cheers Dom.